### ISSUE TITLE
Have you ever thought of building in `localStorage` and `sessionStore`?

### ISSUE THREAD

**Issue by ayangweb** [valtiojs/valtio-persist#10]
Hey 👋 , `getStorage` requires handwriting a set of storage logic which is more cumbersome. Have you ever thought of building in `localStorage` and `sessionStore`?

**Comment by Noitidart** [comment]
That's true. We haven't, but it sound be great to ship some engines, that way engines can get tested by the community to.

**Comment by ayangweb** [comment]
> That's true. We haven't, but it sound be great to ship some engines, that way engines can get tested by the community to.

What can I do for you? I'll tell you what I can do for you.

**Comment by overthemike** [comment]
@ayangweb We're soon going to lauch valtio-persist v2.0.0 which has localstorage and sessionstorage bundled with it (localStorage is actually the default storage strategy). It's up here on github right now, just not yet on npm.

**Comment by overthemike** [comment]
v2.0.3 has been released and contains support for both localStorage and sessionStorage
